releases-istio/apps/products-search-helm-release.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: nordmart-dev-products-seaarch
```

Comment:
Typo in `nordmart-dev-products-seaarch` . Also, let's just keep it as `nordmart-dev-search`

Reply:
Changed